src/main/kotlin/org/opensearch/commons/alerting/model/ClusterMetricsInput.kt

```diff
@@ -15,6 +16,14 @@ import java.io.IOException
 import java.net.URI
 import java.net.URISyntaxException
 
+/**
+ * This regex asserts that the string:
+ *  Starts with a lowercase letter, or digit
+ *  Contains a sequence of characters followed by an optional colon and another sequence of characters
+ *  The sequences of characters can include lowercase letters, uppercase letters, digits, underscores, or hyphens
+ *  The total length of the string can range from 1 to 255 characters
+ */
+val CLUSTER_NAME_REGEX = Regex("^(?=.{1,255}$)[a-z0-9]([a-zA-Z0-9_-]*:?[a-zA-Z0-9_-]*)$")
```

Comment:
Let's remove this and reference the one in IndexUtils

Reply:
Oh good catch. Will do.